py/persistentcode.h

```diff
@@ -31,7 +31,7 @@
 #include "py/emitglue.h"
 
 // The current version of .mpy files
-#define MPY_VERSION 5
+#define MPY_VERSION 6
```

Comment:
Since we are bumping the version here, is it a good time to look at #5807 and/or any other outstanding pull requests that require breaking changes to the .mpy file format?

Reply:
Yes!  There are a few things to do before the next release, to take advantage of this version bump.

Reply:
Did you mean #5807, or something else (maybe to do with dynruntime.h)?

Reply:
I just mentioned that one since it was easy to find because I created it. I didn't search further for others.